src/pages/rechner/[frage].tsx

```diff
@@ -111,15 +112,17 @@ const Frage: NextPage<Props> = ({
 };
 
 export const getStaticPaths: GetStaticPaths = async () => {
+  const questions = await getQuestions();
+
   return {
-    paths: Questions.map(({ id }) => ({ params: { frage: id } })),
+    paths: questions.map(({ id }) => ({ params: { frage: id } })),
     fallback: false,
```

Comment:
Would be better to use `fallback: 'blocking'` and handle the 404-page in `getStaticProps` so that you don't have to redeploy the entire project everytime you add or remove a question from firestore.